idaes/power_generation/unit_models/boiler_heat_exchanger.py

```diff
@@ -300,57 +446,65 @@ def _set_geometry(self):
         self.delta_elevation = Var(
             initialize=0,
             within=NonNegativeReals,
-            doc='Elevation increase used for static pressure calculation - m',
-            units=pyunits.m)
+            doc="Elevation increase used for static pressure calculation - m",
+            units=pyunits.m,
+        )
 
         # Number of tube columns in the cross section plane
         # perpendicular to shell side fluid flow (y direction)
-        self.tube_ncol = Var(initialize=10.0,
-                             within=PositiveReals,
-                             doc='Number of tube columns')
+        self.tube_ncol = Var(
+            initialize=10.0, within=PositiveReals, doc="Number of tube columns"
+        )
 
         # Number of tube rows in the direction of shell side
         # fluid flow (x direction)
-        self.tube_nrow = Var(initialize=10.0,
-                             within=PositiveReals,
-                             doc='Number of tube rows')
+        self.tube_nrow = Var(
+            initialize=10.0, within=PositiveReals, doc="Number of tube rows"
+        )
 
         # Number of inlet tube rows
-        self.nrow_inlet = Var(initialize=1,
-                              within=PositiveReals,
-                              doc='Number of inlet tube rows')
+        self.nrow_inlet = Var(
+            initialize=1, within=PositiveReals, doc="Number of inlet tube rows"
+        )
 
         # Length of a tube in z direction for each path
-        self.tube_length = Var(initialize=5.0,
-                               within=PositiveReals,
-                               doc='Tube length - m',
-                               units=pyunits.m)
+        self.tube_length = Var(
+            initialize=5.0, within=PositiveReals, doc="Tube length - m", units=pyunits.m
+        )
 
         # Inner diameter of tubes
-        self.tube_di = Var(initialize=0.05,
-                           within=PositiveReals,
-                           doc='Inner diameter of tube - m',
-                           units=pyunits.m)
+        self.tube_di = Var(
+            initialize=0.05,
+            within=PositiveReals,
+            doc="Inner diameter of tube - m",
+            units=pyunits.m,
+        )
 
         # Thickness of tube
-        self.tube_thickness = Var(initialize=0.005,
-                                  within=PositiveReals,
-                                  doc='Tube thickness - m',
-                                  units=pyunits.m)
+        self.tube_thickness = Var(
+            initialize=0.005,
+            within=PositiveReals,
+            doc="Tube thickness - m",
```

Comment:
Minor comment, but I would avoid listing units in the doc strings now that we have formal definition of units in the component. It is effectively duplicating the information, and adds a point where things can become inconsistent.